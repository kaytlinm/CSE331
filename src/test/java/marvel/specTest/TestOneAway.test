#Tests that a path can be found between two nodes that have intermediate connections (1 single edge)

CreateGraph graph
AddNode graph B
AddNode graph C
AddNode graph D
AddEdge graph B C e

FindPath graph B C