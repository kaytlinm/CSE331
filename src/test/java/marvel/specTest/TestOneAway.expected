#Tests that a path can be found between two nodes that have intermediate connections (1 single edge)

created graph graph
added node B to graph
added node C to graph
added node D to graph
added edge e from B to C in graph

path from B to C:
B to C via e